src/token.rs

```diff
@@ -85,6 +96,8 @@ impl SgfToken {
                         coordinate,
                     })
             }),
+            "HA" => Some(SgfToken::Handicap(value.parse()
+                .expect(&format!("Error parsing the handicap value : {}", value)))),
```

Comment:
Instead of using `.expect` here which would abort the program, I think we should return a `SgfToken::Invalid` token.